### PR TITLE
Fix badges.

### PR DESCRIPTION
**Pull request by Yeti-or**
💅

### CODE REVIEW

**Comment by Yeti-or** [comment]
But I didn't understand why appveyor is `blond/bem-walk` not `bem-sdk/bem-walk`?

**Comment by blond** [comment]
> But I didn't understand why appveyor is blond/bem-walk not bem-sdk/bem-walk?

shields.io requires login instead of organization name for AppVeyor.

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ bem-walk
 [![Travis Status][test-img]][travis]
 [![Windows Status][appveyor-img]][appveyor]
 [![Coverage Status][coverage-img]][coveralls]
-[![Dependency Status][coverage-img]][coveralls]
+[![Dependency Status][david-img]][david]
```
Comment (blond) [inline README.md]: david -> dependencies

**Comment by blond** [comment]
Closed in favor of https://github.com/bem-sdk/bem-walk/commit/33b0dc28188ecc48362189b5246b6fac22425233

**Pull request closed without merging**